examples/Generator/test_trainer.py

```diff
@@ -21,32 +21,21 @@ class ExampleTrainer(InverseAFMtrainer):
     def on_sample_start(self):
         self.randomize_distance(delta=0.2)
         self.randomize_tip(max_tilt=0.3)
-        # print('Distance, tipR0:', self.distAboveActive, self.afmulator.tipR0)
 
 env = oclu.OCLEnvironment( i_platform = 0 )
 FFcl.init(env)
 oclr.init(env)
 
-args = {
-    'pixPerAngstrome'   : 8,
-    'lvec'              : np.array([
-                            [ 0.0,  0.0, 0.0],
-                            [20.0,  0.0, 0.0],
-                            [ 0.0, 20.0, 0.0],
-                            [ 0.0,  0.0, 4.0]
-                            ]),
-    'scan_dim'          : (128, 64, 20),
-    'scan_window'       : ((2.0, 2.0, 5.0), (18.0, 10.0, 7.0)),
-    'iZPP'              : 8,
-    'QZs'               : [ 0.1,  0, -0.1, 0 ],
-    'Qs'                : [ -10, 20,  -10, 0 ],
-    'amplitude'         : 1.0,
-    'df_steps'          : 10,
-    'initFF'            : True
-}
-
-afmulator = AFMulator(**args)
-afmulator.npbc = (0,0,0)
+afmulator = AFMulator(
```

Comment:
I think `lvec` is missing, no?

Reply:
`lvec` is now an optional argument that gets automatically inferred from the scan window if it's not specified. The behaviour changed since the test was last updated.

Reply:
ok, all good then.